tests/common.py

```diff
@@ -547,7 +549,7 @@ def check_load_and_delete_dataset(dataset_id: str):
 
 
 def create_dummy_dataset_with_collecter_env_helper(
-    dataset_id: str, env: DataCollectorV0, num_episodes: int = 10
+    dataset_id: str, env: DataCollectorV0, num_episodes: int = 10, **kwargs
```

Comment:
what is the purpose of the `kwargs`?

Reply:
It is needed for this new test; they are a general kwarg that can be passed to the`create_dataset_from_collector_env` function  https://github.com/Farama-Foundation/Minari/blob/c5c88e8a03921ac8012a632c741916560670acc2/tests/utils/test_get_normalized_score.py#L20-L27